.github/workflows/repo_file_for_test.repos

```diff
@@ -7,12 +7,12 @@ repositories:
   ament_cmake:
     type: git
     url: https://github.com/ament/ament_cmake.git
-    version: 0.8.1
+    version: 1.3.11
   ament_lint:
     type: git
     url: https://github.com/ament/ament_lint.git
-    version: 0.19.1
+    version: 1.3.11
   ament_package:
     type: git
     url: https://github.com/ament/ament_package.git
-    version: 0.8.1
+    version: 0.16.4
```

Comment:
Similar to my other comment, it should be updated to the last version/tag for Humble.